Makefile.builtem

```diff
@@ -63,7 +65,7 @@ else ifneq ($(filter macos-clang,$(PLATFORM)),)
 endif
 
 # fail for unknown platforms
-PLATFORMS = linux-gcc-64 linux-gcc-32 linux-clang-64 linux-clang-32
+PLATFORMS = linux-gcc-64 linux-gcc-32 linux-gcc-aarch64 linux-clang-64 linux-clang-32
```

Comment:
Note that this is inconsistent: `macos` now uses `arm64`, and linux uses `linux-gcc-aarch64`.

Rust toolchains use `aarch64` for both, flagging this as yu may wish me to change this.